src/realm/sync/instructions.hpp

```diff
@@ -345,6 +370,7 @@ struct Payload {
             switch (lhs.type) {
                 case Type::Erased:
                     return true;
+                case Type::List:
```

Comment:
nit: [[fallthrough]];

Reply:
We don't use [[fallthrough]]; when the labels point to the same code. I collapsed even more cases.